examples/Custom Widget - Hello World.ipynb

```diff
@@ -771,21 +771,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
If you'd like I can go back through them for you and do the conversion, just let me know.